### ISSUE TITLE
Sparse_categorical_crossentropy vs categorical_crossentropy

### ISSUE THREAD

**Issue by JunhoPang** [JunhoPang/keras#1]
https://datascience.stackexchange.com/questions/41921/sparse-categorical-crossentropy-vs-categorical-crossentropy-keras-accuracy

If your targets are one-hot encoded, use categorical_crossentropy. Examples of one-hot encodings:

[1,0,0]
[0,1,0] 
[0,0,1]

But if your targets are integers, use sparse_categorical_crossentropy. Examples of integer encodings (for the sake of completion):

1
2
3
